### PR TITLE
Integrate AsssetChanges Getty API for stored/cached assets

### PR DESCRIPTION
**Pull request by dacox**

### CODE REVIEW

File: pygetty/assets.py [hunk 1]
```diff
@@ -45,3 +46,43 @@ def individual_asset(
...
+):
+    assert asset_type in asset_formatters
+
+    auth_token_manager = flex_auth(
+        auth_token_manager=auth_token_manager,
+        api_key=api_key,
+        client_secret=client_secret,
+    )
+    params = deepcopy(query_params)
+    params['ids'] = ','.join(map(str, ids))
```
Comment (klardotsh) [inline pygetty/assets.py:69]: Isn't it generally recommended (or at least the most readable) to use a comprehension over `map`?

`','.join(ids)` should suffice anyway - `join` always returns a string (which will auto-cast the ideas). Failing that, `','.join(str(id) for id in ids)` I guess?

File: pygetty/util.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def gen_url(*args):
     join_args = [base_url]
     join_args.extend(args)
 
-    return '/'.join(join_args)
+    return '/'.join(map(str, join_args))
```
Comment (klardotsh) [inline pygetty/util.py:13]: ditto here

File: pygetty/util.py [hunk 3]
```diff
@@ -20,4 +20,4 @@ def gen_v3_url(*args):
     join_args = [base_url_v3]
     join_args.extend(args)
 
-    return '/'.join(join_args)
+    return '/'.join(map(str, join_args))
```
Comment (klardotsh) [inline pygetty/util.py:23]: and here

**Review by klardotsh — APPROVED** [review]
Can merge without addressing these nits

**Review by klardotsh — CHANGES_REQUESTED** [review]
WAIT I TAKE IT ALL BACK

You'll need to bump the version number in `pygetty/__init__.py` or `flit` will, rightfully, fail to deploy over an existing version.

File: pygetty/assets.py (hunk 1 above)
Comment (dacox) [reply pygetty/assets.py:69]: @klardotsh for simple methods I tend to think `map` is more readable
However, good to note that `join` demands string arguments:
```
TypeError: sequence item 0: expected str instance, int found
```

Comment (klardotsh) [reply pygetty/assets.py:69]: Interesting - the goofy subtle differences between `join` implementations across languages always trip me up (ex. in JS it's totally legit to just... not pre-cast)

**Review by klardotsh — APPROVED** [review]

**Pull request merged**